components/badge.css

```diff
@@ -0,0 +1,71 @@
+.badge_red{
+    background-color: var(--badge-red);
+}
+
+.badge_green{
+    background-color: var(--badge-green)
+}
+
+.avatar_circle__badge{
+    position: relative;
+}
+
+.badge{
+    position: absolute;
+    border-radius: 50%;
+    border: 1px solid #fff;
+}
+
+.avatar_circle__badge.lg .badge{
+    height: 1.5rem;
+    width: 1.5rem; 
+    bottom: 1.25rem;
+    right: 1.5rem;
+}
+
+.avatar_circle__badge.md .badge{
+    height: 1.25rem;
+    width: 1.25rem; 
+    bottom: 1.25rem;
+    right: 1.15rem;
+}
+
+.avatar_circle__badge.sm .badge{
+    height: 1rem;
+    width: 1rem; 
+    bottom: 1rem;
+    right: 1rem;
+}
+
+.avatar_circle__badge.xs .badge{
+    height: 10px;
+    width: 10px; 
+    bottom: 12px;
+    right: 12px;
+}
+
+.icon_badge{
+    position: relative;
+}
+
+.icon_badge .badge{
+    height: 1rem;
+    width: 1rem; 
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 10px;
+    color: #fff;
```

Comment:
This color can also be put in a variable

Reply:
done!